src/config.js

```diff
@@ -50,15 +47,18 @@ const config = convict({
     format: {
       doc: 'Format to output logs in',
       format: ['ecs', 'pino-pretty'],
-      default: isProduction ? 'ecs' : 'pino-pretty',
+      default: 'pino-pretty',
```

Comment:
for the prod/dev defaults, should these be moved out to an .env file? this force us to be more explicit in whats being set (i.e. setting `LOG_FORMAT` in cdp-app-config env defaults)

Reply:
Yeah I have these in an `.env` file locally. But `.env` file not committed. See `.env.template` in this PR

Also see the PR description for a checkbox to  `Add defaults to cdp-app-config environments rather than programatically deciding them in config in apps`